core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/InlineProjections.java

```diff
@@ -157,8 +161,10 @@ private static Expression inlineReferences(Expression expression, Assignments as
         return inlineSymbols(mapping, expression);
     }
 
-    private static Set<Symbol> extractInliningTargets(ProjectNode parent, ProjectNode child, Session session, TypeAnalyzer typeAnalyzer, TypeProvider types)
+    private static Set<Symbol> extractInliningTargets(PlannerContext plannerContext, ProjectNode parent, ProjectNode child, Session session, TypeAnalyzer typeAnalyzer, TypeProvider types)
```

Comment:
This should be added in the commit that actually needs it, as it's part of that logical change. Otherwise, it looks like a spurious change.

Reply:
While i could make all the changes in one commit, i wanted to make a change that's reviewable.
as a reviewer, i do recommend separating long & boring refactors from more substantial, logical changes.

Reply:
Added `Will be used used soon.` to the commit message.